crypto/armor.go

```diff
@@ -151,7 +152,7 @@ func encryptPrivKey(privKey crypto.PrivKey, passphrase string) (saltBytes []byte
 	}
 
 	key = crypto.Sha256(key) // get 32 bytes
-	privKeyBytes := privKey.Bytes()
+	privKeyBytes := legacy.Cdc.Amino.MustMarshalBinaryBare(privKey)
```

Comment:
Why use the legacy amino codec here?

Reply:
bytes before was the amino encoding. I wasn't sure if I should work on the migration of keys in this PR as its already large.